extensions/google-translate/src/supportedLanguages.json

```diff
@@ -70,8 +70,7 @@
   },
   "Mandarin": {
     "code": "zh-TW",
-    "name": "Taiwanese Mandarin",
-    "flag": "🇹🇼"
+    "name": "Chinese Traditional",
```

Comment:
```suggestion
    "name": "Chinese Traditional"
```

Reply:
Please fix this JSONs and we will be good to go

Reply:
> Please fix this JSONs and we will be good to merge I think
> 
> (you can just approve my change suggestions)

Thank you. I didn't realize those tiny errors.